diagnostic.md

```diff
@@ -15,23 +15,29 @@ Place your responses inside the fenced code-blocks where indivated by comments.
   join table with references to `Movies` and `Profiles`.
 
   ```md
-    # < Your Response Here >
+    In this example, The diagram would be Profiles(many) <==>(many) Favorites(many) <==> (many)Movies.  In this instance, A profile could have multiple entries in the favorite tables concerning movies that it likes while a movie could also show multiple times favorited by other users.
```

Comment:
Yup, we were looking for a table but I realize it wasn't clear from the instructions